### ISSUE TITLE
compile error on elixir latest 1.13.12

### ISSUE THREAD

**Issue by michelson** [elixirs/faker#447]
Hi I got errors on Elixir latest, on 1.12 runs ok

```
#9 1.851 ==> faker
#9 1.851 Compiling 102 files (.ex)
#9 4.111 
#9 4.111 == Compilation error in file lib/faker/string.ex ==
#9 4.111 ** (SyntaxError) lib/faker/string.ex:144:15: invalid bidirectional formatting character in string: \u202A. If you want to use such character, use it in its escaped \u202A form instead
#9 4.111     (elixir 1.13.2) lib/kernel/parallel_compiler.ex:346: anonymous fn/5 in Kernel.ParallelCompiler.spawn_workers/7
#9 4.127 could not compile dependency :faker, "mix compile" failed. Errors may have been logged above. You can recompile this dependency with "mix deps.compile faker", update it with "mix deps.update faker" or clean it with "mix deps.clean faker"
```

**Comment by anthonator** [comment]
What version of faker are you using? This was fixed in #434 and released in 0.17.0.

**Comment by michelson** [comment]
Oh, sorry. Yes, I'm using 0.16; I didn't note the version update.
